docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,98 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
+
+Rationale
+---------
+
+There is often the case where a set of documentation has been built,
+and it is kept online but grows outdated over time.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in Google,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow project maintainers to request the *delisting* of forks that are old and outdated.
+
+Specification
+-------------
+The use cases covered by this policy are:
+
+Outdated docs
+    Documentation that is not being actively maintained or updated.
+
+Unofficial forks
+    Forks of a project documentation that are not linked to from the project in any way.
+
+Implementation
+--------------
+
+Unofficial projects
+~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unofficial* when ALL of the following are met:
+
+* It is not hosted on an official domain of the project
+* It is not linked from the project's README, description, or source code
+
+Unmaintained projects
+~~~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unmaintained* when ALL of the following are met:
+
+* The project is only serving content from releases and commits of a project over 6 months old
+
+Reachability
+~~~~~~~~~~~~
+
+for matters concerning projects that the user owns.
+In every case where contacting the user is necessary,
+the core team will reach out using one of the following means of contact:
+
+* E-mail address on file in the user's profile
+* E-mail address on the GitHub user account where the project is pointed
+
+The core team will stop trying to reach the user after 2 weeks and the user
+will be considered *unreachable*.
```

Comment:
I remember @benjaoming mentioning in another thread about people going on vacation and not being able to reply, which is usually 2 weeks, so maybe increase this to 3 weeks?

Reply:
This is a good point about time frame. I'd probably match our abandoned project policy at 6 weeks even.

I'm also +1 on flipping this policy to be more reactive.

What I would like out of this policy is to be able to take immediate action on a project and be done with the process (in almost all cases). Given we're only talking about blocking the `robots.txt` file to avoid indexing a project, this seems doable.

That is, I'm hoping for a process where we:

- Receive a request about a project
- We determine the verification status of the project in question
- If the project fails verification, we block indexing for the project
- We notify the maintainers about the change we made
- We should never get a negative response from the maintainers of the project in question, but if we do we can review the change

This would compliment the abandoned project policy, but would still allow us to make an immediate change, and we don't have to worry about multiple rounds of follow up.

Reply:
I kind of like 3 weeks more... it means that we can always decide to wait for longer in cases where it would seem more important to reach the project owner.

Reply:
> I assume that you mean maintainers of the official project?

I was describing contacting the maintainers of the unofficial project, to let them know we've determined their project is unofficial and we've altered their project's robots.txt

> By "verification status" did you mean deciding the questions of "unofficial" and "unmaintained"?

Yeah correct -- however we want to refer to that internally :+1:, perhaps verification sounds rather official.

Reply:
Yea, I was leaning this direction when I originally wrote it as well. I think delisting it with a notification is a reasonable policy here. Given that we aren't taking an unreversable action, if they get back to us within a couple days almost nothing will have changed. 

I'm +1 on delisting with notification, instead of a waiting period.

Reply:
Roger and agree :+1: Will implement :hourglass_flowing_sand:

Reply:
Implemented a proposal for this. Please have a look at the new steps.